src/System.Windows.Forms/src/System/Windows/Forms/TabControl.ControlCollection.cs

```diff
@@ -19,13 +19,13 @@ public ControlCollection(TabControl owner) : base(owner)
 
             public override void Add(Control? value)
             {
-                if (value is not TabPage)
+                value.OrThrowIfNull(nameof(value));
+
+                if (value is not TabPage tabPage)
                 {
                     throw new ArgumentException(string.Format(SR.TabControlInvalidTabPageType, value?.GetType().Name));
```

Comment:
```suggestion
                    throw new ArgumentException(string.Format(SR.TabControlInvalidTabPageType, value.GetType().Name));
```
you already checked `value` for `null` on line 22

Reply:
`OrThrowIfNull` does not have the annotations which indicate the input is not null if returned. So replaced with `ArgumentNullException.ThrowIfNull`.